intermediate_source/pinmem_nonblock.py

```diff
@@ -0,0 +1,429 @@
+# -*- coding: utf-8 -*-
+"""
+A guide on good usage of `non_blocking` and `pin_memory()` in PyTorch
```

Comment:
We need to use .rst syntax:
- double backticks instead of single ones
- Links should be `Text <link>`_ not []()

Reply:
Good point, I was planning on doing that - currently the script is a mere export from a regular ipynb, I should make a second pass and correct all the links etc!